app/templates/partials/-result-metadata-help.hbs

```diff
@@ -9,22 +9,22 @@
 
 <h5 class="content-header">Start Time</h5>
 <p>
-  This denotes the time Bullet received the query. This is useful to help you determine whether the data you were looking for had or had not reached Bullet yet.
+  The start time of the query is stored with the key <strong>{{settings.defaultValues.queryCreationTime}}</strong>. This denotes the time Bullet received the query. This is useful to help you determine whether the data you were looking for had or had not reached Bullet yet.
```

Comment:
<strong>{{settings.defaultValues.queryCreationTime}}</strong> is the start time of the query.